lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -96,36 +119,60 @@ feature_nv_overlay_evpn:
 
 maximum_paths:
   kind: int
-  config_get_token_append: '/^maximum-paths (\d+)$/'
-  config_set_append: '<state> maximum-paths <num>'
+  cli_nexus:
+    config_get_token_append: '/^maximum-paths (\d+)$/'
+    config_set_append: '<state> maximum-paths <num>'
+  cli_ios_xr:
+    # BT: Is "maximum-paths" without specialization equal to "maximum-paths ebgp"?
+    config_get_token_append: '/^maximum-paths ebgp (\d+)$/'
+    config_set_append: ' <state> maximum-paths ebgp <num>'
   default_value: 1
 
 maximum_paths_ibgp:
   kind: int
   config_get_token_append: '/^maximum-paths ibgp (\d+)$/'
-  config_set_append: '<state> maximum-paths ibgp <num>'
+  config_set_append: ' <state> maximum-paths ibgp <num>'
   default_value: 1
 
 network:
   multiple: true
-  config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
-  config_set_append: '<state> network <network> <route_map>'
+  cli_nexus:
+    config_get_token_append: '/^network (\S+) ?(?:route-map )?(\S+)?$/'
+    config_set_append: '<state> network <network> <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/^network (\S+) ?(?:route-policy )?(\S+)?$/'
+    config_set_append: ' <state> network <network> <route_policy>'
   default_value: []
 
 next_hop_route_map:
+  _exclude: [cli_ios_xr]
+  kind: string
+  default_value: ""
   config_get_token_append: '/^nexthop route-map (.*)$/'
-  config_set_append: '<state> nexthop route-map <route_map>'
+  config_set_append: ' <state> nexthop route-map <route_map>'
+
+next_hop_route_policy:
+  _exclude: [cli_nexus]
+  kind: string
   default_value: ""
+  config_get_token_append: '/nexthop route-policy (.*)$/'
+  config_set_append: ' <state> nexthop route-policy <route_policy>'
 
 redistribute:
   multiple: true
-  config_get_token_append: '/^redistribute (\S+ ?\S+?) ?(?:route-map (\S+))?$/'
-  config_set_append: '<state> redistribute <protocol>'
+  config_set_append: ' <state> redistribute <protocol>'
+  cli_nexus:
+    config_get_token_append: '/^redistribute (\S+ ?\S+?) ?(?:route-map (\S+))?$/'
+  cli_ios_xr:
+    config_get_token_append: '/^redistribute (\S+ ?\S+?) ?(?:route-policy (\S+))?$/'
   default_value: []
 
 redistribute_policy:
   # route-map/policy is optional on some platforms, required on others
-  config_set_append: '<state> redistribute <protocol> route-map <policy>'
+  cli_nexus:
+    config_set_append: '<state> redistribute <protocol> route-map <policy>'
+  cli_ios_xr:
+    config_set_append: ' <state> redistribute <protocol> route-policy <policy>'
```

Comment:
I think these sets could be combined very easily. In the nu_setter - you'd format the string and pass that as the token. E.g instead of passing <policy> you'd pass <policy-string> equiv where it is route-map for nexus and route-policy for xr. We've done that in several places already.